src/protocol.h

```diff
@@ -26,14 +28,12 @@
 class CMessageHeader
 {
 public:
-    static constexpr size_t MESSAGE_START_SIZE = 4;
```

Comment:
Defining this as `std::tuple_size_v<MessageStartChars>` seems like it would've touched fewer lines